ckb/src/main/java/org/nervos/ckb/type/cell/CellWithStatus.java

```diff
@@ -2,6 +2,8 @@
 
 /** Copyright © 2019 Nervos Foundation. All rights reserved. */
 public class CellWithStatus {
+  public static final String LIVE = "live";
```

Comment:
suggest creating an enum for status: live/dead/unknown

Reply:
Updated. https://github.com/nervosnetwork/ckb-sdk-java/pull/252/commits/c2871cfe874d0ca6966309dcc2d43afd2d145421